friendly-frogs/friendly_frogs/morning/interfaces.py

```diff
@@ -94,3 +94,13 @@ class WeatherInterface:
     wind_deg: Optional[int]
     time: datetime
     clouds: Optional[int]  # percent
+
+
+@dataclass
+class RedditInterface:
+    title: str
+    url: str
+    content: str
+    time_created_unix: int
```

Comment:
Should we maybe have all times in datetime to be more consistent? I used datetime for Weather, but I guess I can switch that to a timestamp as well.